clang/test/dpct/cmake_migation/case_004/expected.txt

```diff
@@ -0,0 +1,14 @@
+target_link_libraries(main PRIVATE cxxopts fmt::fmt -qmkl)
+
+target_link_libraries(transformer_engine PUBLIC 
+   -qmkl
+
+
+
+
+   libdnn.so)
```

Comment:
Have you verified that mapping works by test case?

Reply:
Yes, Done